es_client.py

```diff
@@ -55,23 +55,23 @@ def get_tweets(client, max_tweets=5000):
     :param client: Elasticsearch client object.
     :return: list of tweet represented as dicts.
     """
-    search = Search(index='tweets').using(client).query('match_all').sort('created_at')
+    search = Search(index='tweets')\
+        .using(client)\
+        .query('match_all')\
+        .sort({'created_at': {'order': 'desc'}})[:max_tweets]
     search.execute()
 
     tweets = []
-    for row in search.scan():
-        if len(tweets) <= max_tweets:
-            created_at = _get_with_default(row, 'created_at')
-            full_text = _get_with_default(row, 'full_text')
+    for hit in search:
```

Comment:
Use list splicing to simplify to `for hit in search[:max_tweets]:` and remove if statement below.